app/Exceptions/DockerNotAvailableException.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Exceptions;
+
+use App\Shell\Environment;
+use App\Shell\Shell;
+use Exception;
+
+class DockerNotAvailableException extends Exception
```

Comment:
I found that even though Docker was installed, it wasn't available to a non-root user without some extra steps. I renamed the exception to reflect that distinction.